tokenomics/elixir.md

```diff
@@ -0,0 +1,31 @@
+# Elixir
+
+Elixir is the long term liquidity solution for Dai, EtherDai and MKR. It is a liquidity pool token of one third DAI, one third liquid staked Ether tokens and one third MKR. Elixir is used to provide sustainable liquidity on ethereum mainnet and all L2 networks. 
+
+{% hint style="warning" %}
+This documentation describes planned functionality and processes that MakerDAO has not yet implemented. Be aware that parts may be inaccurate or out of date.
+{% endhint %}
+
+## Endgame Interactions
+
+Maker Core accumulates Elixir using a share of its surplus capital. This Elixir is used to burn MKR in bursts, when the MKR price is favourable.
+
+SubDAOs also accumulate Elixir using their surplus capital. SubDAOs may lock their Elixir. The relative amount of Elixir locked by each SubDAO determines their share of MKR emissions from Maker Core. The more locked Elixir a SubDAO holds, the more MKR emissions it recieves.
+
+SubDAOs use locked Elixir to burn a small percentage of their governance token each year if the value of their locked Elixir exceeds their token's market cap. This delivers value back to the SubDAO's token holders.
+
+Responsibility for Elixir comes under the Protocol Engineering Scope.
```

Comment:
Maybe flesh out what "responsibility" means here? Like are we saying "rules for Elixir specifications and determined by the Protocol Engineering Scope" or more like "maintaining the resiliency of the Elixir product is tasked to the Protocol Engineering Scope."

Reply:
I think I'll actually just take this line out. Not directly relevant imo.